docs/contents/build-stages.md

```diff
@@ -0,0 +1,52 @@
+# Controller Build Stages
+
+We a number of service controller ***build stages***:
+
+* [PROPOSED](#proposed)
+* [PLANNED](#planned)
+* [BUILD](#build)
+* [RELEASED](#released)
+
+## PROPOSED
+
+The `PROPOSED` stage indicates that someone has expressed interest in
+supporting said service in ACK.
+
+There will be one or more Github Issues and/or a Github Project for tracking
+the build of the ACK service controller for the service.
+
+The GitHub Issues **WILL NOT** be associated with a GitHub Milestone.
+
+## PLANNED
+
+The `PLANNED` status indicates that **the AWS service is on our radar** for
+building into ACK.
+
+There will be one or more Github Issues and/or a Github Project for tracking
+the build of the ACK service controller for the service.
+
+There will be a GitHub Milestone that tracks progress towards the `RELEASE`
+build stage for the controller.
+
+## BUILD
+
+The `BUILD` status indicates that the ACK service controller for the AWS
+service is **actively being built** in preparation for a `PREVIEW` release of
+that ACK service controller.
+
+It is in the `BUILD` status that we identify those **AWS service API
+resources** that will be exposed/supported by the controller.
```

Comment:
... in the current release version » basically we reserve the right to modify this in the future. How do we handle change requests?